memegen/services/image.py

```diff
@@ -22,6 +23,13 @@ def find_template(self, key):
     def create_image(self, template, text, style=None):
         image = Image(template, text, style=style)
 
+        data = {
+            'path': text.path,
+            'template': template.key,
+            'style': style,
+        }
+        redis.hmset('latest', data)
```

Comment:
Is there a reason we write to redis in the service, but read from redis in the store?